CHANGES.md

```diff
@@ -11,8 +11,9 @@
   * Do not use `object` in class inheritance as this is default behaviour in Python3
   * Move formatting in `reporters.py` into `Reporter` and `DeltaReporter` base classes to have a uniform formatting: new `stringify` in `utils.py` method and corresponding test in `test_stringify.py`. The subclasses and only inherit from these two base classes to ensure a consistent formatting across the reporters
   * Change datastructure of coverage information from `namedtuple` (immutable) to `dictionary` (mutable) in `reporters.py`. Adjusted `html-delta.jinja2`, `html.jinja2` and `filters.py` accordingly. This change in datastructure leads to a more compact and more readable code
-  * `pycobertura show` and `pycobertura diff` now include json format
-  * `pycobertura show` and `pycobertura diff` now include markdown format
+  * `pycobertura show` and `pycobertura diff` now include `json` format
+  * `pycobertura show` and `pycobertura diff` now include `markdown` format
+  * `pycobertura show` and `pycobertura diff` now include `yaml` format
```

Comment:
It's looking verbose again, haha. Can we do:

```suggestion
  * `pycobertura show` and `pycobertura diff` now support output formats: `json`, `markdown`, and `yaml`
```

Reply:
done in https://github.com/aconrad/pycobertura/pull/131/commits/3c8ed0d7fb565f8875a927f9571a9068137086d2